opte/src/engine/icmpv6.rs

```diff
@@ -349,6 +349,8 @@ impl HairpinAction for RouterAdvertisement {
             dst: meta.inner_ip6().unwrap().src,
             proto: Protocol::ICMPv6,
             next_hdr: IpProtocol::Icmpv6,
+            // RFC 4861 7.1.2 requires that the hop limit be 255 in an RA.
+            hop_limit: 255,
```

Comment:
Nit, but 7.1.2 is about validating NAs. The section specifying this for RAs is 6.1.2.

Reply:
Thanks, good catch. Looks like the guest I was using (alpine + dhcpd) doesn't check but we should be setting the same for NAs too.